app/assets/stylesheets/move_to_ui_gem.scss

```diff
@@ -29,3 +29,9 @@ hr.dashed {
         padding-left: 26px;
     }
 }
+
+tr {
```

Comment:
Could you add `tr#order-notes-table` to this? Or are there other `actions` classes that all need the same styling?